frigate/video.py

```diff
@@ -176,6 +176,7 @@ def capture_frames(
         try:
             frame_buffer[:] = ffmpeg_process.stdout.read(frame_size)
         except Exception as e:
+            fps.value = 0
```

Comment:
I think it would be better to set this in the `CameraWatchdog` instead since sometimes this thread hangs and it might not reach this line.